doc/user_guide.md

```diff
@@ -0,0 +1,324 @@
+# User guide
+
+The guide will go over the deployment of the Exasol Azure helper Functions briefly and then focus on using them in azure data factory itself.
+
+## Requirements
+
+An azure subscription with sufficient privileges to create resource groups, storage accounts, function apps and an azure data factory.
+
+## Deployment
+
+A full section on all deployment options available for Azure Functions can be found in the azure function docs under the how-to guides: [Zip push deployment for Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/deployment-zip-push)
+
+In this guide I'll elaborate on 1 of the easiest user friendly ways to set up azure functions. 
+
+The microsoft documentation on deploying azure functions is way more exhaustive so if this doesn't cover your needs you might find a better option there.
+
+Let's get started:
+
+### Zip Deployment using the Azure Portal and Azure CLI:
+
+Setting everything up in the azure portal:
+
+Navigate to the azure portal and log in if you haven't done so yet: [Microsoft Azure](https://portal.azure.com/#home)
+
+On the dashboard, click on 'Create a resource'.
+
+![](./img/2021-08-04-15-02-42-image.png)
+
+Search for 'function', then pick 'Function App'.
+
+![](./img/2021-08-04-15-04-40-image.png)
+
+Click 'Create'.
+
+![](./img/2021-08-04-15-05-43-image.png)
+
+Now we need to do some setup:
+
+![](./img/2021-08-04-15-06-27-image.png)
+
+Create a new resource group (or use an existing one if you want). Take note of the name.
+
+Pick a (unique) function app name.
+
+Pick '.NET' as the runtime stack and pick '3.1' as the version.
+
+Pick the region that fits you the most (this will probably the same region as where you host the Azure Data Factory as well).
+
+![](./img/2021-08-04-15-08-55-image.png)
+
+Next, press 'Hosting'. This will bring you to the next setup screen.
+
+![](./img/2021-08-04-15-11-27-image.png)
+
+Create a new storage account name (recommended) or use an existing one. 
+
+Take note of the name.
+
+As operating system pick 'Windows', ()this is because we use a .dll from the Exasol ADO .NET data provider that's not crossplatform.)
```

Comment:
```suggestion
As operating system pick 'Windows' (this is because we use a .dll from the Exasol ADO .NET data provider that's not cross-platform).
```